FAQ.md

```diff
@@ -381,3 +380,11 @@ it can [cause
 problems](https://groups.google.com/d/msg/golang-nuts/AnMr9NL6dtc/UnyUUKcMCAAJ).
 If your dependers are using `dep`, this is not a concern, as `dep` takes care of
 stripping out nested `vendor` directories.
+
+## How do I configure a dependency that doesn't tag its releases?
+
+Add a constraint to your manifest that specifies `branch: "master"`
+for the dependency. `dep` will deduce the current revision number of your
+dependencies master branch, and place it the lock-file (`Gopkg.lock`) for you.
+Anyone who clones your project will, therefore, wind up with same version of the dependency,
```

Comment:
I'm not sure this is quite what we want to be saying here. Committing `vendor` is what would guarantee this _at clone time_; having the rev in your `Gopkg.lock` only guarantees that `vendor` will be populated with it by `dep ensure` (assuming no other changes).

Just the last sentence, referring the reader to the other question, is probably sufficient.